<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanSegmentation1Class</class>
 <widget class="QMainWindow" name="HumanSegmentation1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HumanSegmentation1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(161, 161, 161);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>216</width>
      <height>384</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Human segmentation</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Load&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ser_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;width  :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ser_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;height :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_info">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>400</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_h">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_w">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Load Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;fps&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_fps_s">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_fps_c">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_ser_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Serial :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ser_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;CUDA :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>label_ser_2</zorder>
    <zorder>label_ser_3</zorder>
    <zorder>pushButton_info</zorder>
    <zorder>textBrowser_h</zorder>
    <zorder>textBrowser_w</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pause</zorder>
    <zorder>label_10</zorder>
    <zorder>label_ser_6</zorder>
    <zorder>textBrowser_fps_s</zorder>
    <zorder>label_ser_4</zorder>
    <zorder>textBrowser_fps_c</zorder>
   </widget>
   <widget class="QPushButton" name="shape_sz">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="input_z">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="input_save">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shape_saveserial">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_shapeserial">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>216</width>
      <height>384</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Shape_serial&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_shapeCUDA">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>216</width>
      <height>384</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="shape_saveCUDA">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Shape_CUDA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shape_cz">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_input</zorder>
   <zorder>groupBox</zorder>
   <zorder>shape_sz</zorder>
   <zorder>input_z</zorder>
   <zorder>input_save</zorder>
   <zorder>shape_saveserial</zorder>
   <zorder>label_shapeserial</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_shapeCUDA</zorder>
   <zorder>shape_saveCUDA</zorder>
   <zorder>label_4</zorder>
   <zorder>shape_cz</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HumanSegmentation1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>LoadSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>LoadSourceVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pause</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>checkpause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_z</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>inputzoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_info</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>show_info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>827</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shape_sz</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>shapezoom_s()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shape_cz</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>shapezoom_c()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1161</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>1091</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_save</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>save_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shape_saveserial</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>save_serial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shape_saveCUDA</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>save_cuda()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>HumanSegmentation1Class</receiver>
   <slot>Hsegmentation_CUDA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>976</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoadSource()</slot>
  <slot>LoadSourceVideo()</slot>
  <slot>ShowVideo()</slot>
  <slot>inputzoom()</slot>
  <slot>jointzoom()</slot>
  <slot>show_info()</slot>
  <slot>checkplay()</slot>
  <slot>shapezoom_s()</slot>
  <slot>shapezoom_c()</slot>
  <slot>save_input()</slot>
  <slot>save_serial()</slot>
  <slot>save_cuda()</slot>
  <slot>Hsegmentation_CUDA()</slot>
 </slots>
</ui>
